CHANGELOG.md

```diff
@@ -2,14 +2,144 @@
 
 Mark  items as `Added`, `Changed`, `Fixed`, `Modified`, `Removed`, `Untested Features`, `Upcoming Features`, `Known Issues`
 
-## 6.1.0
+
+
+## 7.0.0 - 'release date'
```

Comment:
Don't forget to add the release date on the day of release

Reply:
Noted 👍